docs/modules/ROOT/pages/tls.adoc

```diff
@@ -9,13 +9,25 @@ See xref:hazelcast:security:tls-ssl.adoc[TLS/SSL Basics] for more detailed infor
 
 == Prerequisites for TLS
 
-Before you create and configure TLS, you need to create Kubernetes secret:
+Before you create and configure TLS, you need to create Kubernetes secret. Kubernetes secret can be created in two different ways:
+
+If you just want to provide certificates to create the keystore:
 
 [source,shell]
 ----
 kubectl create secret tls example --cert=example.crt --key=example.key
```

Comment:
we should use `kubectl create secret generic` for all options to prevent confusion.

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/234/commits/29d69be1ddfbc6eef110e906f8959f0d82588f28